sdk/include/reactos/shdocvw_undoc.h

```diff
@@ -44,6 +44,30 @@ WinList_RegisterPending(
     _In_ DWORD dwUnused,
     _Out_ PLONG plCookie);
 
+/*****************************************************************************
+ * IConnectionPointCB interface (Unconfirmed)
```

Comment:
unconfirmed or undocumented?
How did you obtain this interface layout?

Reply:
I have got this by inspecting the virtual table.
These method names are entirely my imagination.

Reply:
Populate the fields of the record (a.k.a. structure).
Mismatched. Retype. Mismatched. Retype...

That's "Trial and Error".